dockerfiles/Dockerfile.dashboard.rpm

```diff
@@ -31,6 +32,8 @@ RUN set -x \
     && mkdir gopath \
     && go env -w GOPROXY=https://goproxy.cn,direct \
     && cd /tmp/ \
+    # add startup script
+    && curl --silent --location https://raw.githubusercontent.com/totemofwolf/apisix-build-tools/service/deps/manager-api.service | tee /tmp/build/output/apisix/dashboard/usr/lib/systemd/system/manager-api.service \
```

Comment:
@totemofwolf  I was wondering that if we should use the `manager-api.service` file within this commit? Thanks.

Reply:
It looks like you have put it in `deps/manager-api.service`, so don't need to download it, just need to `mv`...

Reply:
> It looks like you have put it in `deps/manager-api.service`, so don't need to download it, just need to `mv`...

Yes u r right. But now I can't find a `suitable` way to do this. ;-)